### DISCUSSION TITLE
How to add Open frequent needed URL

### DISCUSSION THREAD

**Discussion by MaheshtheDev** [Flow-Launcher/Flow.Launcher#1226] [Q&A]
I have some URLs, I open them regularly. Is there any way, that I can access those URLs with a few strokes

**Comment by jjw24** [comment]
Hey how's it going?

At the moment there isn't a plugin that offers to save URLs, this might be a good simple plugin to add to the Plugin Store. The closest to this would be using the Bookmark plugin, but will involve saving those URLs as bookmarks.

**Comment by kubalav** [comment]
I can suggest plugin [Favorites](https://github.com/stax76/Flow.Launcher.Plugin.Favorites). Configuration is a bit tricky, but plugin works as expected.

**Comment by jjw24** [comment]
This could be a feature we can add in either for Bookmark plugin (as a custom saved/bookmarked urls only in flow) or the URL plugin.

I have occasionally also found there are certain urls I want to access quickly but not necessarily want to bookmark them.

Thoughts @Flow-Launcher/team ?

**Reply by taooceros** [reply]
I agree for the URL plugin. Maybe we can save accessed url?

**Reply by jjw24** [reply]
yeah just save it to settings json

**Reply by jjw24** [reply]
instead of saving all accessed urls, maybe just a context menu option to save it manually.

**Reply by deefrawley** [reply]
Why not just allow static web pages in the web searches plugin i.e. - no search term required?

**Reply by jjw24** [reply]
Sorry what do you mean by allow? As in save them with WebSearch plugin?

**Reply by deefrawley** [reply]
> Sorry what do you mean by allow? As in save them with WebSearch plugin?

Yeah - define them next to search URLS but if no search term {q} is defined then have them launch straight after the shortcut (I tried creating a URL with no search term but it wouldn't trigger until the keyword had something after it in Flow)

**Reply by jjw24** [reply]
Hmm I think plugin responsibility wise, URL plugin is a better fit than WebSearch plugin I feel. WebSearch is for conducting actual searches with provided address where as a straight forward url access just need to be opened.
